consent/models.py

```diff
@@ -239,3 +245,159 @@ class Meta(Consent.Meta):
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+    NOTIFICATION_MESSAGE_HELP_TEXT = mark_safe_lazy(_(
+        'Fill in a text for message of the notification which appears on dashboard '
+        'when user decline to provide consent. '
+        'The following variables may be available:<br />'
+        '<ul>'
+        '<li>enterprise_customer_name: A name of enterprise customer.</li>'
+        '<li>course_title: Title of course. Available when type of consent is course.</li>'
+        '</ul>'
+    ))
+
+    page_title = models.CharField(
+        max_length=255,
+        help_text=_('Title of page')
+    )
+    left_sidebar_text = models.TextField(
+        null=True,
+        blank=True,
+        help_text=SIDEBAR_PARAGRAPH_HELP_TEXT
+    )
+    top_paragraph = models.TextField(
+        null=True,
+        blank=True,
+        help_text=TOP_PARAGRAPH_HELP_TEXT
+    )
+    agreement_text = models.TextField(
+        null=True,
+        blank=True,
+        help_text=_('Text next to agreement check mark')
+    )
+    continue_text = models.CharField(
+        max_length=255,
+        help_text=_('Text of agree button')
+    )
+    abort_text = models.CharField(
+        max_length=255,
+        help_text=_('Text of decline link')
+    )
+    policy_dropdown_header = models.CharField(
+        max_length=255,
+        null=True,
+        blank=True,
+        help_text=_('Text of policy drop down')
+    )
+    policy_paragraph = models.TextField(
+        null=True,
+        blank=True,
+        help_text=POLICY_PARAGRAPH_HELP_TEXT
+    )
+    confirmation_modal_header = models.CharField(
+        max_length=255,
+        help_text=_('Heading text of dialog box which appears when user decline to provide consent')
+    )
+    confirmation_modal_text = models.TextField(
+        help_text=CONFIRMATION_MODAL_HELP_TEXT
+    )
+    modal_affirm_decline_text = models.CharField(
+        max_length=255,
+        help_text=_('Text of decline button on confirmation dialog box')
+    )
+    modal_abort_decline_text = models.CharField(
+        max_length=255,
+        help_text=_('Text of abort decline link on confirmation dialog box')
+    )
+    declined_notification_title = models.TextField(
+        help_text=NOTIFICATION_TITLE_HELP_TEXT
+    )
+    declined_notification_message = models.TextField(
+        help_text=NOTIFICATION_MESSAGE_HELP_TEXT
+    )
+    enterprise_customer = models.OneToOneField(
+        EnterpriseCustomer,
+        related_name="data_sharing_consent_page",
+        on_delete=models.deletion.CASCADE
+    )
+    published = models.BooleanField(
+        default=False,
+        help_text=_(
+            _("Specifies whether data sharing consent page is published.")
+        )
+    )
+    history = HistoricalRecords()
```

Comment:
We are trying to remove all uses of `django-simple-history`. Please remove it from this model.